src/Morphic-Widgets-Basic/PluggableButtonMorph.class.st

```diff
@@ -976,24 +945,18 @@ PluggableButtonMorph >> mouseDown: evt [
 { #category : #'event handling' }
 PluggableButtonMorph >> mouseEnter: evt [
 	"Update the appearance."
+
 	self setProperty: #mouseEntered toValue: true.
-	
-	self gradientLook
-		ifTrue: [self changed]
-		ifFalse: ["0.09375 is exact in floating point so no cumulative rounding error will occur"
-				self color: (self color adjustBrightness: -0.09375)].
```

Comment:
This line should not be removed

Reply:
Can you explain the idea behind this line?

Reply:
I tried to re-add this line but I don't see any difference in the behaviour. But since you seems to know more than me on this stuff I'll trust you :)